docs/CONFIGURATION.md

```diff
@@ -44,3 +44,7 @@ Environment variables use the prefix `KNPT_` followed by the configuration path
 | `KNPT_MITZ_TLSKEYFILE`              | `mitz.tlskeyfile`              | Path to private key (only for .pem certs)                                                                                                                                                                                                                     |
 | `KNPT_MITZ_TLSKEYPASSWORD`          | `mitz.tlskeypassword`          | Password for .p12/.pfx                                                                                                                                                                                                                                        |
 | `KNPT_MITZ_TLSCAFILE`               | `mitz.tlscafile`               | Path to server certificate                                                                                                                                                                                                                                    |
+| **Tracing / OpenTelemetry**         |                                |                                                                                                                                                                                                                                                               |
```

Comment:
Do we need some additional documentation, e.g. in "docs/DEPLOYMENT.md" on tracing?